core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -5530,6 +5530,58 @@ private void checkSemiJoinRuleOnAntiJoin(RelOptRule rule) {
         .checkUnchanged();
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5035">[CALCITE-5035]
+   * Pull up constant project under sort</a>. */
+  @Test void testSortProjectPullUpConstants1() {
+    // The constant's count is more than sort's field, and without any fetch or offset
+    final String sql = "select e.empno, e.ename, e.deptno from sales.emp e\n"
```

Comment:
Nitpick:
```suggestion
    final String sql = "select e.empno, e.ename, e.deptno\n"
        + "from sales.emp e\n"
```
If you accept this change, please be consistent in the other tests.

PS: I see that in the test, both styles are used, but what I suggest seems preferred for long lines like in your case, up to you :)